src/app/pizarra/pizarra.component.ts

```diff
@@ -25,7 +25,7 @@ export class PizarraComponent implements OnInit {
   }
   private filtrarTareas(): void {
 
-    this.tareas$ = this.tareasService.getTareas$();
+    this.tareas$ = this.tareasService.tareas$;
 
     this.tareasPendientes$ = this.tareas$
```

Comment:
aqui puedes hacer esto: this.tareasPendientes$ = this.tareasService.tareas$
      .pipe(map(val => val.filter(dat => dat.status === TareaStatus.PENDIENTE)));